rust/src/detect/transforms/hash.rs

```diff
@@ -35,7 +35,6 @@ static mut G_TRANSFORM_SHA256_ID: c_int = 0;
 
```

Comment:
should this commit reference
https://redmine.openinfosecfoundation.org/issues/7498
?

Reply:
Yep.